test/hooks/validate-animal.test.js

```diff
@@ -0,0 +1,33 @@
+
+const assert = require('assert');
+const feathers = require('@feathersjs/feathers');
+const validateAnimal = require('../../src/hooks/validate-animal.js');
+
+describe('\'validate animal\' hook', () => {
+  let app;
+
+  beforeEach(() => {
+    app = feathers();
+
+    app.use('/animals', {
+      async create(data) {
+        return data;
+      },
+      async get(id) {
+        return { id: '1234', name: 'dog'};
```

Comment:
I can (try) to update this to use the actual schema (if it is necessary) 🐕

Reply:
This is a test for the hook. You can replace this json to match the schema.